benchmarks/opacities_opacity.py

```diff
@@ -28,7 +29,7 @@ class BenchmarkMontecarloMontecarloNumbaOpacities(BenchmarkBase):
         }
     )
     def time_compton_opacity_calculation(self, electron_number_density, energy):
-        calculate_opacity.compton_opacity_calculation(
+        tardis.opacities.compton_opacity_calculation.compton_opacity_calculation(
```

Comment:
`import as` perhaps?

Reply:
Or `from x import compton_opacity_calculation`

Reply:
this should not have moved